README.md

```diff
@@ -16,7 +16,7 @@ BitShares Core
 * [License](#license)
 
 BitShares Core is the BitShares blockchain implementation and command-line interface.
-The web wallet is [BitShares UI](https://github.com/bitshares/bitshares-ui).
+The browser web based wallet is [BitShares UI](https://github.com/bitshares/bitshares-ui).
```

Comment:
"The browser web based wallet" seems a bit strange. I've seen "web" wallets on the Internet, also "web-based" wallets, "browser-based" wallets, and probably "web browser based" wallets, but didn't see one "browser web based" wallet :P

Reply:
I meant web browser based :P